test/RestTest.php

```diff
@@ -116,4 +116,27 @@ function($sKey, $aDefaultValue) use ($aParameters) {
 		$this->assertEquals(['retcode' => 0], $oRestClient->ListOperations());
 	}
 
```

Comment:
I suggest to move this function somewhere else to mock RestClient and be able to test this new feature. for example we could do it in Utils in a static call (not the best solution). here is a possible way:
`
	/**
	 * @var string Keeps track of the latest date the datamodel has been installed/updated
	 * (in order to check which modules were installed with it)
	 */
	protected static $sLastInstallDate;

	/**
	 * Check if the given module is installed in iTop.
	 * Mind that this assumes the `ModuleInstallation` class is ordered by descending installation date
	 *
	 * @param string $sName Name of the module to be found
	 * @param bool $bRequired Whether to throw exceptions when module not found
	 * @param \RestClient|null $oClient: provider only in test conditions
	 *
	 * @return bool True when the given module is installed, false otherwise
	 * @throws Exception When the module is required but could not be found
	 */
	public static function CheckModuleInstallation(string $sName, bool $bRequired = false, RestClient $oClient = null): bool
	{
		if (is_null($oClient)){
			$oClient = new RestClient();
		}

		try {
			if (!isset(static::$sLastInstallDate)) {
				$aDatamodelResults = $oClient->Get('ModuleInstallation', ['name' => 'datamodel'], 'installed', 1);
				if ($aDatamodelResults['code'] != 0 || empty($aDatamodelResults['objects'])){
					throw new Exception($aDatamodelResults['message'], $aDatamodelResults['code']);
				}
				$aDatamodel = current($aDatamodelResults['objects']);
				static::$sLastInstallDate = $aDatamodel['fields']['installed'];
			}

			$aResults = $oClient->Get('ModuleInstallation', ['name' => $sName, 'installed' => static::$sLastInstallDate], 'name,version', 1);
			if ($aResults['code'] != 0 || empty($aResults['objects'])) {
				throw new Exception($aResults['message'], $aResults['code']);
			}
			$aObject = current($aResults['objects']);
			Utils::Log(LOG_DEBUG, sprintf('iTop module %s version %s is installed.', $aObject['fields']['name'], $aObject['fields']['version']));
		} catch (Exception $e) {
			$sMessage = sprintf('%s iTop module %s is considered as not installed due to: %s', $bRequired ? 'Required' : 'Optional', $sName, $e->getMessage());
			if ($bRequired) {
				throw new Exception($sMessage, 0, $e);
			} else {
				Utils::Log(LOG_INFO, $sMessage);
				return false;
			}
		}
		return true;
	}
`